### ISSUE TITLE
Steps to train and play against AI

### ISSUE THREAD

**Issue by bupticybee** [deeptexas-ai/MasterAI-1.0-1vs1-Limit#1]
Hi, this is a pretty new and interesting project. However I didn't see any description about how the train and play against the agent trained.

Could you provide the step to train the agent and hardware requirements? Thanks in advance

Or should I say Is this a "real" opensourced project or just an advertise?

**Comment by masterai-top** [comment]
1. Hardware platform for AI training: 
CPU: 128-core, RAM: 512GB, HD: 2T 
2. Output Model can be provided upon request
3. AI training application can be provided upon request
Are you an AI developer?

**Comment by bupticybee** [comment]
So it's a advertise after all.

**Comment by masterai-top** [comment]
> Hi, this is a pretty new and interesting project. However I didn't see any description about how the train and play against the agent trained.
> 
> Could you provide the step to train the agent and hardware requirements? Thanks in advance
> 
> Or should I say Is this a "real" opensourced project or just an advertise?

1. Hardware platform for AI training: 
CPU: 128-core, RAM: 512GB, HD: 2T 
2. Output Model can be provided upon request
3. AI training application can be provided upon request
Are you an AI developer?

**Comment by masterai-top** [comment]
> So it's a advertise after all.

1. Hardware platform for AI training: 
CPU: 128-core, RAM: 512GB, HD: 2T 
2. Output Model can be provided upon request
3. AI training application can be provided upon request
Are you an AI developer?

**Comment by bupticybee** [comment]
> > So it's a advertise after all.
> 
> 1. Hardware platform for AI training:
>    CPU: 128-core, RAM: 512GB, HD: 2T
> 2. Output Model can be provided upon request
> 3. AI training application can be provided upon request
>    Are you an AI developer?

Yes, I am a RL researcher in shenzhen,china. Check my github page you will find some project strongly related. I have to say this project is wired, there is no "Install" or "Usage" or anything like that to guide people how to use it.

Not sure why you opensource this project and didn't write any documentation.

I am requesting model and training procedure.

Why am I even speaking English here.... I see chinese comments in your code, assuming your team is located in china?

And BTW, it's rude to close an issue without reply.
